markdown templates/arm-template/quickstart-resource-manager-template.md

```diff
@@ -7,7 +7,7 @@ ms.service: azure-resource-manager
 ms.topic: quickstart
 ms.custom: subject-armqs
 ms.author: your-msft-alias
-ms.date: 03/17/22
+ms.date: 07/16/20
```

Comment:
Thanks for the submission, but the changes to the **quickstart-resource-manager-template.md** are not accepted. Please do not merge the changes.